Romo/src/Selection/RMActivityChooserRobotController.m

```diff
@@ -124,19 +124,19 @@ - (void)showRomoControlAlert
         messageTemplate = NSLocalizedString(@"RomoControl-Message-NonCompatible-Device", @"Visit http://romo.tv on another local iDevice to control me.");
     }
     
-    NSString *romoNumber = [[RMTelepresencePresence sharedInstance] number];
-    
-    if ([UIDevice currentDevice].isDockableTelepresenceDevice && !romoNumber.length) {
-        [[RMTelepresencePresence sharedInstance] fetchNumber:^(NSError *error) {
-            if (error) {
-                [[[RMAlertView alloc] initWithTitle:NSLocalizedString(@"Error", @"Error")
-                                            message:NSLocalizedString(@"RomoControl-Message-Number-Error", @"There was an issue fetching my Romo number. Make sure you are connected to the internet.")
-                                           delegate:nil] show];
-            } else {
-                [self showRomoControlAlert];
-            }
-        }];
-    }
+    NSString *romoNumber = @"123-456-7890"; //[[RMTelepresencePresence sharedInstance] number];
```

Comment:
Should this be hardcoded?

Reply:
These are part of OpenTok TelepresencePresence. The existing library does not have ARM64 symbols, and latest library is too large to put on git and is incompatible with some methods used by Romo. Since there's little value maintaining OpenTok code, I though it might be better to remove them from the project, or keep them but do not compile them.